src/components/forgot-password-drawer/ForgotPasswordDrawer.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+
+import { StyleSheet, View } from 'react-native'
+
+import IconQuestionMark from 'app/assets/images/iconQuestionMark.svg'
+import Drawer from 'app/components/drawer'
+import Text from 'app/components/text'
+import { useDrawer } from 'app/hooks/useDrawer'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { useThemeColors, ThemeColors } from 'app/utils/theme'
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    mainView: {
+      display: 'flex',
+      flexDirection: 'column',
+      paddingTop: 24,
+      paddingHorizontal: 24,
+      fontFamily: 'AvenirNextLTPro-Regular'
+    },
+
+    headerView: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'center',
+      alignItems: 'center',
+      marginBottom: 24
+    },
+
+    questionMarkIcon: {
+      marginRight: 12
+    },
+
+    title: {
+      fontSize: 18,
+      fontWeight: '800',
+      lineHeight: 22,
+      color: themeColors.neutral
+    },
+
+    bodyView: {
+      display: 'flex',
+      flexDirection: 'column'
+    },
+
+    restoreAccount: {
+      fontWeight: '700',
+      fontSize: 16,
+      lineHeight: 24,
+      color: themeColors.neutral,
+      textAlign: 'center',
+      marginBottom: 24
+    },
+
+    emailView: {
+      display: 'flex',
+      flexDirection: 'row',
+      borderWidth: 1,
+      borderColor: themeColors.neutralLight6,
+      borderRadius: 8,
+      padding: 12,
+      marginBottom: 45
+    },
+
+    emailSubHeadings: {
+      display: 'flex',
+      flexDirection: 'column'
+    },
+
+    fromHeader: {
+      color: themeColors.neutralLight4,
+      fontWeight: 'bold',
+      fontSize: 16,
+      lineHeight: 20,
+      marginBottom: 12
+    },
+
+    subjectHeader: {
+      color: themeColors.neutralLight4,
+      fontWeight: 'bold',
+      fontSize: 16,
+      lineHeight: 20
+    },
+
+    emailContent: {
+      display: 'flex',
+      flexDirection: 'column',
+      paddingLeft: 24,
+      flexShrink: 1
+    },
+
+    fromContent: {
+      color: themeColors.neutral,
+      fontWeight: '600',
+      fontSize: 16,
+      lineHeight: 20,
+      marginBottom: 12
+    },
+
+    subjectContent: {
+      color: themeColors.neutral,
```

Comment:
general thought here for us, it might be helpful for us to have some established typography, so we can build off those and make sure our designs use consistent values, what do you think? i have a few ideas on how we can represent it

Reply:
@milindNshah You shouldn't have to specify the color as neutral, the Text component does that by default

@dylanjeffers It would be ideal for sure, but this is another one of those cases where our designs aren't necessarily consistent enough to make common patterns valuable. But maybe having the common patterns would push us towards more consistent designs. For now, the `Text` component sets the fontFamily + color

Reply:
Thanks for the context!